homeassistant/components/fibaro/binary_sensor.py

```diff
@@ -75,9 +75,9 @@ def __init__(self, fibaro_device: DeviceModel) -> None:
             self._attr_icon = SENSOR_TYPES[self._fibaro_sensor_type][1]
 
     @property
-    def extra_state_attributes(self) -> Mapping[str, Any] | None:
```

Comment:
Is `None` definitely needed?

Reply:
None is not needed and therefore I removed it.